_data/xcodes.yml

```diff
@@ -1,4 +1,43 @@
 osx_images:
+  - image: xcode9
+    xcode: "9.0"
+    osx_version: "10.12"
+    xcode_full_version: "9 beta 2"
```

Comment:
Since it's named **9 beta 2** on https://developer.apple.com/download/ I thought following that would make the most sense